hkmc2/shared/src/main/scala/hkmc2/bbml/bbML.scala

```diff
@@ -13,6 +13,7 @@ import semantics.*, semantics.Term.*
 import Elaborator.Ctx
 import syntax.*
 import Tree.*
+import codegen.js.Scope
```

Comment:
Also don't forget to move `Scope` out of the `hkmc2.codegen.js` package and into `hkmc2.utils`.